packages/neuron-wallet/tests/block-sync-renderer/indexer-cache-service.intg.test.ts

```diff
@@ -88,7 +101,7 @@ const mockGetTransactionHashes = (mocks: any[] = []) => {
 
   for (const lock of [formattedDefaultLockScript, formattedAcpLockScript, formattedLegacyAcpLockScript]) {
     const { hashes } = mocks.find(mock => mock.lock === lock) || { hashes: [] }
-    stubbedConstructor.calledWith(expect.anything(), { lock }).mockReturnValue({
+    stubbedConstructor.calledWith(expect.anything(), { lock }, undefined, { includeStatus: false }).mockReturnValue({
```

Comment:
It's better to mock the `this.indexer.ckbRpcUrl` as a string, but not undefined.